src/applications/personalization/dashboard-2/tests/e2e/benefits-of-interest.cypress.spec.js

```diff
@@ -16,30 +21,6 @@ function sectionHeadingsExist() {
   }).should('exist');
 }
 
-// Helper to make sure that the "health care" info does or doesn't exist
-function healthCareInfoExists(exists) {
```

Comment:
moved these functions to the shared `helpers` file so they can be reused in the new `not-in-mpi-error` tests